docs/manager/README.md

```diff
@@ -1145,20 +1137,22 @@ All errors will be displayed in standard output and logged into `/var/log/clwpos
     ```
     Consider the example for checking alt-php74 redis.so:
     ```
-    ls /opt/alt/php74/usr/lib64/php/modules | grep redis.so
+    # ls /opt/alt/php74/usr/lib64/php/modules | grep redis.so
     redis.so
     ```
-6. If the Redis module is missing:
-   7. install the `alt-phpXY-pecl-ext` package manually
-   8. run the Redis configuration script `/usr/share/cloudlinux/wpos/enable_redis_for_alt_php.py` 
+2. If the Redis module is missing:
+   
+   Install the `alt-phpXY-pecl-ext` package manually
```

Comment:
These are steps (install, then run), could you please turn them into list, either a numbered one, or a bulleted one?

Reply:
done